src/controllers/__tests__/sessionLockController.test.ts

```diff
@@ -24,17 +24,75 @@ const mockSetBlockedUser = jest
 const mockUnsetBlockedUser = jest
   .fn()
   .mockImplementation(async () => E.right(true));
+const mockUserHasActiveSessions = jest
+  .fn()
+  .mockImplementation(async () => E.right(true));
 const mockRedisSessionStorage = ({
   delUserAllSessions: mockDelUserAllSessions,
   setBlockedUser: mockSetBlockedUser,
-  unsetBlockedUser: mockUnsetBlockedUser
+  unsetBlockedUser: mockUnsetBlockedUser,
+  userHasActiveSessions: mockUserHasActiveSessions
 } as unknown) as RedisSessionStorage;
 
 const mockDel = jest.fn().mockImplementation(async () => E.right(true));
 const mockRedisUserMetadataStorage = ({
   del: mockDel
 } as unknown) as RedisUserMetadataStorage;
 
+describe("SessionLockController#getUserSession", () => {
+  it("should fail on invalid fiscal code", async () => {
+    const req = mockReq({ params: { fiscal_code: "invalid" } });
+    const res = mockRes();
+
+    const controller = new SessionLockController(
+      mockRedisSessionStorage,
+      mockRedisUserMetadataStorage
+    );
+
+    const response = await controller.getUserSession(req);
+    response.apply(res);
+
+    expect(res.status).toHaveBeenCalledWith(400);
+  });
+
+  it("should fail on get error", async () => {
```

Comment:
Is more than a single get, maybe a more specific description remove ambiguity for example `should fail if userHasActiveSessions  returns an error`

Reply:
you are right. thank you!